erigon-lib/txpool/fetch.go

```diff
@@ -177,7 +177,7 @@ func (f *Fetch) receiveMessage(ctx context.Context, sentryClient sentry.SentryCl
 	var req *sentry.InboundMessage
 	for req, err = stream.Recv(); ; req, err = stream.Recv() {
 		if err != nil {
-			f.logger.Error("[txpool.receiveMessage]", "err", err)
+			f.logger.Debug("[txpool.receiveMessage]", "err", err)
```

Comment:
no reason to log ctx cancelation. 
for example if node shutting-down by Ctrl+C - it's not an error, but expected behavior. 
let's move under `select` or let's use `errors.Is(context.Canceled)`

Reply:
return wrapped error instead of logging